### DISCUSSION TITLE
Is it possible to configure ssr:false at build time?

### DISCUSSION THREAD

**Discussion by misbahansori** [nuxt/nuxt#27519] [Questions] [answered]
Hi, I have a nuxt app with capacitor installed, and its working fine. But one thing to note is that I have to set `ssr:false` in nuxt.config.js to make it work on capacitor. But I want to use the `ssr:true` for the web version. 

Is it possible to configure `ssr:false` at build time? So that I can build the app for capacitor with `ssr:false` and for the web with `ssr:true`. 

Thanks in advance.

**Comment by AloisSeckar** [comment, marked as answer]
I would try introducing env variable like `NUXT_USE_SSR` and then in config `ssr: process.env.NUXT_USE_SSR`. Nuxt should be able to read the env variable correctly during build time. So then you just have to set it accordingly before each variant of your build.

**Reply by misbahansori** [reply]
Hi, thanks for the answer. I didn't think about that. I thought that `process.env` only works for `runtimeConfig`. it turns out that `process.env` also works anywhere in the nuxt.config.js file. I've tried it and it works. Thanks a lot.

**Reply by TheAlexLichter** [reply]
@misbahansori just be aware that you *can not change it* after the build process when using `process.env.XYZ` in your nuxt config. In your case, that works.
